app/javascript/packages/rails-i18n-webpack-plugin/extract-keys-webpack-plugin.js

```diff
@@ -78,16 +78,16 @@ class ExtractKeysWebpackPlugin {
     compiler.hooks.compilation.tap('compile', (compilation) => {
       compilation.hooks.additionalAssets.tapPromise(PLUGIN, () =>
         Promise.all(
-          compilation.chunks.map((chunk) =>
+          [...compilation.chunks].map((chunk) =>
```

Comment:
I feel like we had issues with the array splat before and switched to `Array.from` ... but that was only in IE right? So since this is a server-side plugin that shouldn't matter?

Reply:
> I feel like we had issues with the array splat before and switched to `Array.from` ... but that was only in IE right? So since this is a server-side plugin that shouldn't matter?

Right, it was IE, and I don't expect it should be an issue here.

Coincidentally I was staring at this Babel configuration earlier today, wondering if it had something to do with those issues we were seeing:

https://babeljs.io/docs/en/assumptions#arraylikeisiterable